File: 7-kyu/number-to-bytes/javascript/solution.js
const toBytes = n => {
	let nBin = n.toString(2);
  while (nBin.length % 8 !== 0) {
    nBin = "0" + nBin;
  }
  const bytes = [];
  for (let i = 0; i < nBin.length; i += 8) {
    const byte = nBin.slice(i, i + 8);
    bytes.push(byte);
  }
  return bytes;
};

File: 7-kyu/number-to-bytes/python/solution.py
def to_bytes(n):
    n_bin = f'{n:b}'
    while len(n_bin) % 8 != 0:
        n_bin = '0' + n_bin
    return [n_bin[i:i + 8] for i in range(0, len(n_bin), 8)]
